internal/utils/copy_from_image.go

```diff
@@ -38,6 +39,58 @@ func CopyFromImage(image, srcInImage, destOnHost string) error {
 	if err != nil {
 		return fmt.Errorf("copy from image: %s: %w", stderr.String(), err)
 	}
+	excludeFiles := []string{".gitkeep", ".ini", ".env", ".DS_Store"}
+	excludeDirs := []string{".git", ".zeabur", ".vscode", ".idea", ".github"}
+	err = deleteFilesRecursively(excludeFiles, destOnHost)
+	if err != nil {
+		return fmt.Errorf("delete files in directory: %w", err)
+	}
+	err = deleteDirectories(excludeDirs, destOnHost)
+	if err != nil {
+		return fmt.Errorf("delete directories in directory: %w", err)
+	}
+
+	return nil
+}
+
+func deleteFilesRecursively(deleteFiles []string, path string) error {
+	err := filepath.Walk(path, func(filePath string, fileInfo os.FileInfo, err error) error {
+		if err != nil {
+			return err
+		}
+
+		if fileInfo.IsDir() {
+			return nil
+		}
+		fileName := fileInfo.Name()
 
+		for _, targetFile := range deleteFiles {
+			if !strings.EqualFold(fileName, targetFile) {
+				continue
+			}
+			filePath := filepath.Join(path, fileName)
```

Comment:
```suggestion
			filePath := filepath.Join(path, targetFile)
```

`fileName.lower() == targetFile.lower()` ≠ `fileName == targetFile`.

Reply:
`fileName` should be the actual filename in real scene. We should delete with `fileName` if the target system is case sensitive. The targetFile is just the match pattern.